js/tools/kermit.js

```diff
@@ -76,6 +79,26 @@ function grep (path, pattern) {
     });
 }
 
+// This hash is derived from sha1('link-table'), and for now is the hardcoded
+// location of link tables for all shards.
+const LINK_TABLE_ID = '4dba9091495e8f277893e0d400e9e092f9f6f551';
```

Comment:
I thought we did the `GLib.Checksum.new` inline, but I just checked domain.js, and I guess not. OK.